packages/core/src/VRange.ts

```diff
@@ -123,6 +106,31 @@ export class VRange {
         }
         return selectedNodes;
     }
+    /**
+     * Return a list of all the nodes currently targeted by the range. If the
+     * range is collapsed, the targeted node is the container of the range.
+     * Otherwise, the targeted nodes are the ones encountered by traversing the
+     * tree from the start to end of this range.
+     */
+    targetedNodes(predicate?: Predicate): VNode[];
+    targetedNodes<T extends VNode>(predicate?: Constructor<T>): T[];
+    targetedNodes<T>(predicate?: Predicate<T>): VNode[];
+    targetedNodes<T>(predicate?: Predicate<T>): VNode[] {
+        if (this.isCollapsed()) {
+            const target = this.start.parent;
```

Comment:
This should target _all_ ancestors, not just the parent. Otherwise quid of nested blocks?